core/src/main/java/org/apache/carbondata/core/scan/filter/executer/RestructureEvaluatorImpl.java

```diff
@@ -104,6 +108,12 @@ protected boolean isDimensionDefaultValuePresentInFilterValues(
     return isDefaultValuePresentInFilterValues;
   }
 
+  @Override
+  public BitSet prunePages(RawBlockletColumnChunks rawBlockletColumnChunks)
+      throws FilterUnsupportedException, IOException {
+    return new BitSet();
```

Comment:
I think for this operation we need to throw `FilterUnsupportedException` similar to applyFilter implementation

Reply:
ok